src/utils/supports.js

```diff
@@ -9,12 +9,13 @@ const containPixelfed = /pixelfed/i;
 const notContainPixelfed = /^(?!.*pixelfed).*$/i;
 const containPleroma = /pleroma/i;
 const containAkkoma = /akkoma/i;
+const notContainPixelfedAkkomaPleroma = /^(?!.*pixelfed|.*akkoma|.*pleroma).*$/i;
```

Comment:
😂 I think, as this is becoming more complex, (simple, negated?) regex check is no longer feasible. Best way is for the check to accept a function. Maybe like:

```js
'@mastodon/trending-hashtags': (version) => !containPixelfed.test(version) && !containPleroma.test(version) && !containAkkoma.test(version)
```

I initially added `notContainPixelfed` because I was lazy and didn't expect to add more conditions 🙈

Reply:
yeah that sounds like the wiser decision lol